.circleci/config.yml

```diff
@@ -1,19 +1,54 @@
 # Python CircleCI 2.1 configuration file
 # for mets-mods2tei
 #
-# Check https://circleci.com/docs/2.1/language-python/ for more details
+# Check https://circleci.com/docs/2.0/language-python/ for more details
 #
 version: 2.1
```

Comment:
Maybe just `2`?

Reply:
Cf. https://circleci.com/docs/2.0/language-python/

Reply:
Both `2.1` and `2` yield a 404, hence the change.

Reply:
Oh sry, you were not referring to the comment, but to the `version` setting, let's see...

Reply:
No, since we rely on the codecov **orb**, we [require](https://circleci.com/docs/2.0/configuration-reference/#version) 2.1